unit_tests/test_install_script.sh

```diff
@@ -249,5 +250,27 @@ testFullConfig(){
   assertEquals 1 $?
 }
 
+### Manage client libraries security config
+testManageClientLibrariesSecurityConfig() {
+  rm $environment_file
```

Comment:
[Small note that this sort of errors, or at least just emits an unneeded log](https://gitlab.ddbuild.io/DataDog/agent-linux-install-script/-/jobs/460233248#L512) - don't expect it to be a blocker/problem, just unexpected for me to see it when I was reviewing the test logs.